pkg/reconciler/v1alpha1/clusteringress/resources/virtual_service.go

```diff
@@ -158,6 +159,14 @@ func makeMatch(host string, pathRegExp string) v1alpha3.HTTPMatchRequest {
 	return match
 }
 
+// hostRegExp returns an ECMAScript regular expression to match either host or host:<any port>
+func hostRegExp(host string) string {
+	// Should only match 1..65535, but for simplicity it matches 0-99999
+	portMatch := "(?::\\d{1,5})?"
+
+	return fmt.Sprintf("^%s%s$", host, portMatch)
```

Comment:
May have to do some quoting/escaping for the Host part to avoid having `.` matching?

Reply:
https://golang.org/pkg/regexp/#QuoteMeta

Reply:
Oh good call, missed that!